src/components/AssetsPage/pageTypes.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import Button from '@edx/paragon/src/Button';
+
+import WrappedAssetsDropZone from '../AssetsDropZone';
+import WrappedAssetsTable from '../AssetsTable';
+import WrappedAssetsFilters from '../AssetsFilters';
+import WrappedPagination from '../Pagination';
+import edxBootstrap from '../../SFE.scss';
+
+export const types = {
+  NORMAL: 'normal',
+  NO_CONTENT: 'noContent',
+  NO_RESULTS: 'noResults',
+};
+
+const renderAssetsDropZone = maxFileSizeMB => (
+  <WrappedAssetsDropZone
+    maxFileSizeMB={maxFileSizeMB}
+  />
+);
+
+export const renderAssetsPage = maxFileSizeMB => (
+  <React.Fragment>
+    <div className={edxBootstrap.col}>
+      { renderAssetsDropZone(maxFileSizeMB) }
+      <WrappedAssetsFilters />
```

Comment:
TODO: I should probably pull this out into a renderAssetsFilters function like the renderAssetsDropZone function above it.